README.md

```diff
@@ -4,22 +4,31 @@
+AWIPS takes a unified approach to data ingest, where most data ingested into the system comes through the [LDM](#ldm) client pulling data feeds from the [Unidata IDD](https://www.unidata.ucar.edu/projects/#idd). Various raw data and product files (netCDF, grib, BUFR, ASCII text, gini, AREA) are decoded and stored as HDF5 files and Postgres metadata by [EDEX](install/install-edex), which serves products and data over http.
 
-* [CAVE](http://unidata.github.io/awips2/install/install-cave) - the **C**ommon **A**WIPS **V**isualization **E**nvironment
-* [python-awips](https://github.com/Unidata/python-awips) - a Python data access framework for requesting Numpy data arrays and Shapely geometries.
+Unidata supports two data visualization frameworks: [CAVE](install/install-cave) (an Eclipse-built Java application which runs on Linux, Mac, and Windows), and [python-awips](python/overview) (a python package).
 
-# License
+> **Note**: Our version of CAVE is a **non-operational** version.  It does not support some features of NWS AWIPS.  Warnings and alerts cannot be issued from Unidata's CAVE.  Additional functionality may not be available as well.
 
-Unidata AWIPS source code and binaries (RPMs) are considered to be in the public domain, meaning there are no restrictions on any download, modification, or distribution in any form (original or modified).  Unidata AWIPS contains no proprietery content and is therefore not subject to export controls as stated in the Master Rights licensing file. 
 
-# AWIPS Data in the Cloud
+![CAVE](https://www.unidata.ucar.edu/software/awips2/images/Unidata_AWIPS2_CAVE.png)
 
-Through a grant provided by [Jetstream](https://jetstream-cloud.org/), Unidata is able to run a real-time EDEX data server in cloud, providing free AWIPS data to UCAR member institutions and other geoscience research and education organizations.  When prompted in the Connectivity Preferences dialog, enter **`edex-cloud.unidata.ucar.edu`** (without adding http:// before, or :9581/services after), or select it from the default dropdown list. 
+---
+
+## License
+
+Unidata AWIPS source code and binaries (RPMs) are considered to be in the public domain, meaning there are no restrictions on any download, modification, or distribution in any form (original or modified).  Unidata AWIPS license information can be found [here](https://github.com/Unidata/awips2/blob/unidata_18.2.1/LICENSE).
+
+---
+
+## AWIPS Data in the Cloud
+
+Unidata and XSEDE Jetstream have partnered to offer an EDEX data server in the cloud, open to the community.  Select the server in the Connectivity Preferences dialog, or enter **`edex-cloud.unidata.ucar.edu`** (without *http://* before, or *:9581/services* after).
+
+![EDEX in the cloud](/images/boEbFSf28t.gif)
```

Comment:
This image link is broken now.

Reply:
Ahhh thank you - a lot of the links were actually broken because the README was one directory up, so I should have fixed all of those as well.